server/tests/testcases.js

```diff
@@ -1223,6 +1223,14 @@ module.exports = [
     `,
     expected: 'p' },
 
+  { name: 'Object.setPrototypeOf(..., null)', src: `
+    var o = {parent: 'o'};
+    var q = Object.create(o);
+    Object.setPrototypeOf(q, null) === q &&
+        Object.getPrototypeOf(q);
```

Comment:
This fits on one line, no?

Reply:
Yes.  Fixed.  I think the test might have at some point had more conditions.